### PR TITLE
[sablier-v1-deposit] refactor: rename Sablier V1 strategy

### PR DESCRIPTION
**Pull request by PaulRBerg**
This PR introduces two changes in the Sablier V1 strategy added in this PR: https://github.com/snapshot-labs/snapshot-strategies/pull/828

1. Rename the strategy to "sablier-v1-deposit" to make it clear that this strategy refers to V1, and not V2 (ref https://github.com/snapshot-labs/snapshot-strategies/pull/1235)
2. Update the subgraph URLs because we changed our GitHub organization name in the meantime from `sablierhq` to `sablier-labs`

### CODE REVIEW

File: src/strategies/sablier-v1-deposit/index.ts [hunk 1]
```diff
@@ -3,16 +3,16 @@ import { formatUnits } from '@ethersproject/units';
...
+  '5': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-goerli', // goerli
+  '10': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-optimism', // optimism
+  '42': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-kovan', // kovan
+  '56': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-bsc', // bsc
+  '137': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-matic', // polygon
+  '42161': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-arbitrum', // arbitrum
+  '43114': 'https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-avalanche' // avalanche
 };
 
 export const author = 'dan13ram';
```
Comment (ChaituVR) [inline src/strategies/sablier-v1-deposit/index.ts:18]:
```suggestion
export const author = 'sablier-labs';
```

File: src/strategies/sablier-v1-deposit/examples.json
Comment (ChaituVR) [inline src/strategies/sablier-v1-deposit/examples.json:1]: @PaulRBerg `subGraphURL` param is failing the test cases

Comment (PaulRBerg) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Thanks @ChaituVR.

@razgraf and @maxdesalle could you please take a look at this? Not sure what's going on

Comment (PaulRBerg) [reply src/strategies/sablier-v1-deposit/examples.json:1]: @ChaituVR sorry for that, it should work now!

Comment (ChaituVR) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Hi @PaulRBerg test cases are still failing, can take a look? i think we need one more address in the example and we are good to go

Comment (PaulRBerg) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Sorry @ChaituVR, fixed. But it looks like CI is still failing 😞

> Errors found in subgraphRequest: URL: https://api.thegraph.com/subgraphs/name/sablier-labs/sablier-goerli, Status: 200,  Response: [{"message":"service is overloaded and can not run the query right now. Please try again in a few minutes"}]

Is it due to The Graph? If yes, can you re-run the CI and see if it works now?

Comment (ChaituVR) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Yea 🙈 Should add addresses that got voting power

Comment (PaulRBerg) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Oooh sorry, I didn't know that.

Looks like I made it work now!

Comment (ChaituVR) [reply src/strategies/sablier-v1-deposit/examples.json:1]: Thanks @PaulRBerg Works well!

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**